docs/docs/60-new-docs/50-user-guide/40-examples.md

```diff
@@ -4,8 +4,30 @@ sidebar_label: Examples
 
 # Examples
 
-:::warning
-This section is expected to be quite extensive.
+Here you'll find practical demonstrations and reference implementations showcasing how to effectively use Kargo in your GitOps workflows.
+
+Whether you're new to Kargo or looking to optimize your existing deployment pipelines, these examples will help you understand key concepts and implement powerful CD workflows. Each example includes detailed explanations, configuration files, and step-by-step guides to get you started.
+
+## Basic
+### Argo CD-Driven Examples
```

Comment:
I think this whole section needs some kind of disclaimer.

See https://docs.kargo.io/new-docs/user-guide/patterns/#pseudo-gitops for explanation.